sonic-chassisd/scripts/chassisd

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env python2
```

Comment:
Please convert to Python 3, as Python 2 is deprecated, and we are in the process of converting all Python code to Python 3

Reply:
Moved to Python3.